docs/mobile-apps/automated-testing/ipa-files.md

```diff
@@ -60,31 +60,26 @@ You can use any of the existing methods of distribution for your iOS app, except
 
 ## Creating an XCUITest Package
 
-:::tip
-Our XCUITest test runner accepts both .app and .ipa file formats for the `--app` and `--test` parameters.
-:::
+`saucectl` supports both .app and .ipa file formats [Testing with XCUITest](/mobile-apps/automated-testing/espresso-xcuitest).
 
-### -Runner.app
+### .app
 
 1. Open your application project in Xcode.
 2. Select **Generic iOS Device** or **Any iOS Device (arm64)** as your project's device target.
 3. Make sure that the your UI tests are part of a **Target Membership** and that those Targets are selected to be built in your Xcode **Build scheme**. _Targets containing UI Tests are typically selected to be built at the "Test" build action._
 <img src={useBaseUrl('img/xcuitest/xcode-build.png')} alt="Xcode Build Options" width="800" />
 
 4. Generate your test package by selecting **Product** > **Build For** > **Testing**.
-5. Navigate to your Xcode project's **Products** directory and find the generated **-Runner.app** files.
-6. Your -Runner.app files are ready to be used in the `--test` parameter of our XCUI test runner.
+5. Navigate to your Xcode project's **Products** directory and find the generated **.app** files.
 
 ### .ipa (Optional)
 
-1. Create an empty directory name `Payload`.
-2. Move the -Runner.app to the `Payload` directory.
-3. Compress the Payload directory into an archive (.zip file) and give it a new name with .ipa file format ending.
-4. Your .ipa file is ready to be used in the `--test` parameter of our XCUITest test runner. See [Real Device Testing with Espresso and XCUITest](mobile-apps/automated-testing/espresso-xcuitest/real-devices.md) for more information.
+1. Create an empty directory name **Payload**.
+1. Move the `<AppName>.app` to the **Payload** directory.
+1. Compress the **Payload** directory into an archive (.zip file) and rename it using .ipa file format ending.
```

Comment:
Since `saucectl` `v0.47.0+` supports archiving from `.app` to `.ipa`, we may remove that, and advertise to use version `v0.47.0+`